pylint/ruff.toml

```diff
@@ -0,0 +1,12 @@
+# This extend our general Ruff rules specifically for tests
```

Comment:
Is that temporary until we can enable the full ruff config ?

Reply:
I don't know, this is just copied from core